### DISCUSSION TITLE
How do i add a title screen?

### DISCUSSION THREAD

**Discussion by Ballerguywhat** [TrianglyRU/OrbinautFramework#46] [Q&A] [answered]
![Screenshot 2024-10-24 211013](https://github.com/user-attachments/assets/34ec0d07-e9a8-4a35-af4f-5087f3f63243)
I want to add a title screen, ive just tried adding my title screen art to rm_devmenu but it doesn't show the full art work. Please help!
![image_2024-10-24_211145204](https://github.com/user-attachments/assets/758d44d1-f705-4554-83f8-bc82baa849b0)

**Comment by TrianglyRU** [comment, marked as answer]
The game's default resolution is 400x224. Having an image with higher resolution than that doesn't make any sense for a menu
